File: 2. Calculations/exercise08/PizzaParty.java
import java.util.Scanner;

public class PizzaParty {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int people, pizzas, slices, slicesPerPerson, leftover;
        String output;

        System.out.print("How many people? ");
        people = scan.nextInt();

        System.out.print("How many pizzas do you have? ");
        pizzas = scan.nextInt();

        System.out.print("How many slices in a pizza? ");
        slices = scan.nextInt();

        scan.close();

        slicesPerPerson = (pizzas * slices) / people;
        leftover = (pizzas * slices) % people;

        output = String.format("\n%d people with %d pizzas"
            + "\nEach person gets %d piece(s) of pizza."
            + "\nThere are %d leftover pieces.", people, pizzas, slicesPerPerson, leftover);

        System.out.println(output);
    }
}

File: 2. Calculations/exercise08/python/pizza_party.py
# Exercise 08 - Pizza Party

# Get inputs from user
num_of_people = input('How many people? ')
num_of_pizzas = input('How many pizzas do you have? ')
slices_per_pizza = input('How many slices in a pizza? ')

# Convert inputs from string to integer
num_of_people = int(num_of_people)
num_of_pizzas = int(num_of_pizzas)
slices_per_pizza = int(slices_per_pizza)

# Perform calculations
slices_per_person = int((num_of_pizzas * slices_per_pizza) / num_of_people)
leftover = (num_of_pizzas * slices_per_pizza) % num_of_people

# Format output
output_text = f'{num_of_people} people with {num_of_pizzas} pizzas' \
              f'\nEach person gets {slices_per_person} piece(s) of pizza.' \
              f'\nThere are {leftover} leftover pieces.'

print(output_text)
